### DISCUSSION TITLE
Support for D7

### DISCUSSION THREAD

**Discussion by CCH44BI** [AliDehbansiahkarbon/ChatGPTWizard#29] [General]
I am sure there are lots of active D7 users out there.

Is it technically possible for ChatGPTWizard to support D7 ?

**Comment by AliDehbansiahkarbon** [comment]
Well, I would like to do that and technically it's possible but there are some time-consuming challenges to deal with, there is no Json library, no Unicode, no Generics, it is impossible to use something like XSuperObject because there is no RTTI library (there is an alternative but not rich), no HTTP client(must use Indy or clever which is not free), no class helpers, weak OpenToolsAPI, and etc.
It will be really time-consuming to adjust everything without third-party components and for free, maybe I will do that if someone pushes(pays) me!🤣
